migrations/20241105164503_2.sql

```diff
@@ -18,12 +18,13 @@ create type underway.task_retry_policy as (
     max_attempts         int,
     initial_interval_ms  int,
     max_interval_ms      int,
-    backoff_coefficient  float
+    backoff_coefficient  float,
+    jitter_factor        float
```

Comment:
Sorry for the delay on this. I think we probably want to incorporate these changes in a separate migration, right? 

If we don't, then anyone using Underway today wouldn't have a clean upgrade path. (Maybe that's reasonable for breaking releases, but it would be nice if we could avoid that.)